capture/Makefile

```diff
@@ -1,7 +1,12 @@
 #VERSION = debug
  VERSION = release
 
-PLATFORM = x64
+#PLATFORM = x86
+ PLATFORM = x64
+#PLATFORM = armv5
+#PLATFORM = armv6
+#PLATFORM = armv7
+#PLATFORM = armv8
```

Comment:
Thanks for adding this. Would be helpful to add a note to the readme mentioning that the right line needs to be uncommented here before running `make`.

Reply:
cba4e98